content/en/blog/_posts/2022-12-01-cpumanager-goes-GA.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: blog
+title: 'Kubernetes 1.26: CPUManager goes GA'
+date: 2022-12-01
+slug: cpumanager-goes-GA
+---
+
+**Author:**
+Francesco Romani (Red Hat)
+
+The CPU Manager is a part of the kubelet, the Kubrnetes node agent, which enables the user to allocate exclusive CPUs to containers.
+Since its [Beta graduation in 2018](blog/2018/07/24/feature-highlight-cpu-manager/), the CPU Manager adoption steadily grown and become a staple
+of performance-critical and low-latency setups.
+
+Since Kubernetes 1.10, the CPU Manager proved itself reliable and fulfilling its role. Over time, changes were about bugfixes or internal refactoring
```

Comment:
```suggestion
Since Kubernetes 1.10, the CPU Manager proved itself reliable and fulfilled its role of allocating exclusive CPUs to containers. Over time, changes were about bugfixes or internal refactoring
```

Reply:
thanks, rephrased taking into account your suggestion.